pkg/cloudcore/apis/config/default.go

```diff
@@ -0,0 +1,75 @@
+package config
+
+import (
+	"path"
+
+	"github.com/kubeedge/kubeedge/common/constants"
+	commonconfig "github.com/kubeedge/kubeedge/pkg/common/apis/config"
+)
+
+func NewDefaultCloudCoreConfig() *CloudCoreConfig {
+	return &CloudCoreConfig{
+		Kube:             NewDefaultKubeConfig(),
+		EdgeController:   NewDefaultEdgeControllerConfig(),
+		DeviceController: NewDeviceControllerConfig(),
+		Cloudhub:         NewDefaultCloudHubConfig(),
+		Modules:          NewDefaultModules(),
+	}
+}
+
+func NewDefaultEdgeControllerConfig() *EdgeControllerConfig {
+	return &EdgeControllerConfig{
+		NodeUpdateFrequency: 10,
+		ControllerContext: &ControllerContext{
+			SendModule:     constants.CloudHubControllerModuleName,
+			ReceiveModule:  constants.EdgeControllerModuleName,
+			ResponseModule: constants.CloudHubControllerModuleName,
+		},
+	}
+}
+
+func NewDeviceControllerConfig() *DeviceControllerConfig {
+	return &DeviceControllerConfig{
+		ControllerContext: &ControllerContext{
+			SendModule:     constants.CloudHubControllerModuleName,
+			ReceiveModule:  constants.DeviceControllerModuleName,
+			ResponseModule: constants.CloudHubControllerModuleName,
+		},
+	}
+}
+
+func NewDefaultCloudHubConfig() *CloudHubConfig {
+	return &CloudHubConfig{
+		EnableWebsocket:    true,
+		WebsocketPort:      10000,
+		EnableQuic:         false,
+		QuicPort:           10001,
```

Comment:
Can we define constant for these values ? It will be easy to resuse if needed and also easy to change